### PR TITLE
Added Game.getOutfits(playerSex) and Game.getMounts()

### PR DESCRIPTION
**Pull request by EPuncker**
Co-Authored-By: Zbizu <Zbizu@users.noreply.github.com>

### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
these will be used by podium description and cyclopedia character view

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

File: src/luascript.cpp [hunk 1]
```diff
@@ -982,6 +982,16 @@ void LuaScriptInterface::pushOutfit(lua_State* L, const Outfit* outfit)
 	setMetatable(L, -1, "Outfit");
 }
 
+void LuaScriptInterface::pushMount(lua_State* L, const Mount* mount)
+{
+	lua_createtable(L, 0, 5);
+	setField(L, "name", mount->name);
+	setField(L, "speed", mount->speed);
+	setField(L, "clientId", mount->clientId);
```
Comment (nekiro) [inline src/luascript.cpp:990]: isnt clientId looktype?

File: src/luascript.cpp [hunk 2]
```diff
@@ -4486,6 +4498,48 @@ int LuaScriptInterface::luaGameGetHouses(lua_State* L)
...
+	if (playerSex > PLAYERSEX_LAST) {
+		lua_pushnil(L);
+		return 1;
+	}
+
+	const auto& outfits = Outfits::getInstance().getOutfits(playerSex);
+	lua_createtable(L, outfits.size(), 0);
+
+	int index = 0;
+	for (auto outfit : outfits) {
```
Comment (nekiro) [inline src/luascript.cpp:4519]: you are copying here, const auto& outfit

File: src/luascript.cpp [hunk 3]
```diff
@@ -4486,6 +4498,48 @@ int LuaScriptInterface::luaGameGetHouses(lua_State* L)
...
+}
+
+int LuaScriptInterface::luaGameGetMounts(lua_State* L)
+{
+	// Game.getMounts()
+	const auto& mounts = g_game.mounts.getMounts();
+	lua_createtable(L, mounts.size(), 0);
+
+	int index = 0;
+	for (auto mount : mounts) {
```
Comment (nekiro) [inline src/luascript.cpp:4535]: same as above

File: src/luascript.cpp (hunk 1 above)
Comment (EPuncker) [reply src/luascript.cpp:990]: `<mount id="1" clientid="368" name="Widow Queen" speed="20" premium="yes" />`
yes?!

Comment (nekiro) [reply src/luascript.cpp:990]: I find it weird to be named clientid, but if its in xml then okay

**Review by nekiro — APPROVED** [review]

**Review by Zbizu — APPROVED** [review]

**Pull request merged**